drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/ValueOptionsPageTest.java

```diff
@@ -297,4 +300,20 @@ public void testCanSetupBindingWhenBindingIsEnabledAndConstraintValueIsNotLitera
 
         assertFalse(page.canSetupBinding());
     }
+
+    @Test
+    public void testValueListDisabledWhenEnumsPresent() throws Exception {
```

Comment:
Hi @karreiro , during writing this test, I found out one interesting thing in the class `ValueOptionsPage`. Please look at these piece of code [1]. I am wondering about the method `factType()` in this piece of code. Is there some special reason for the `factType()` method (maybe in whole class)? I mean, would be there some problem if we replace all occurrences of `factType()` by `plugin().getFactType()`.

[1]
https://github.com/kiegroup/drools-wb/blob/master/drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/ValueOptionsPage.java#L304-L307